astropy/tests/helper.py

```diff
@@ -461,52 +460,24 @@ def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
     :func:`numpy.testing.assert_allclose`.
     """
     import numpy as np
+    from ..units.quantity import _unquantify_allclose_arguments
     np.testing.assert_allclose(*_unquantify_allclose_arguments(actual, desired,
                                                                rtol, atol),
                                **kwargs)
 
 
+@deprecated("3.1", alternative='astropy.units.allclose')
 def quantity_allclose(a, b, rtol=1.e-5, atol=None, **kwargs):
     """
     Returns True if two arrays are element-wise equal within a tolerance.
 
     This is a :class:`~astropy.units.Quantity`-aware version of
     :func:`numpy.allclose`.
     """
-    import numpy as np
-    return np.allclose(*_unquantify_allclose_arguments(a, b, rtol, atol),
-                       **kwargs)
+    from ..units.quantity import allclose as quantity_allclose_units
+    quantity_allclose_units(a, b, rtol=1.e-5, atol=None, **kwargs)
 
 
 def _unquantify_allclose_arguments(actual, desired, rtol, atol):
```

Comment:
This one should be removed altogether. Nobody should be relying on this private routine.

Reply:
You mean to remove the function `quantity_allclose` altogether?
Shouldn't a deprecated function call to the new location of the function?

Reply:
No, please look where the comment is made: you should remove `_unquantity_allclose_arguments` from `utils` -- it is a private function, so no deprecation is needed.

Reply:
You meant to remove from `units` not `utils`, right? If you do mean units, isn't that the place I moved the function `_unquantity_allclose_arguments` to?Also, this comment is made in tests/helper.py.Should I remove it from `helper` as well or keep it as it is.

Reply:
I meant `utils` - you moved `_unquantity_allclose_arguments` to `units` and there is no reason to keep it here, as you import it directly from units in `assert_quantity_allclose`.